src/Tomcat/TomcatController.ts

```diff
@@ -263,9 +259,9 @@ export class TomcatController {
                 }
             });
 
-            const javaProcess: Promise<void> = Utility.executeCMD(output, 'java', { shell: true }, ...this.getJavaArgs(serverInfo, true));
             this.startDebugSession(serverInfo);
-            await javaProcess;
+            setTimeout(() => this.setStarted(serverInfo, true), 500);
+            await Utility.executeCMD(output, 'java', { shell: true }, ...this.getJavaArgs(serverInfo, true));
```

Comment:
Emm, I prefer starting the java process first.

Reply:
yes, misunderstood the await in javascript.